addons/l10n_kr/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name': "Korea",
+    'icon': '/account/static/description/l10n.png',
+    'countries': ['kr'],
+    'version': '1.0',
+    'category': 'Localization',
+    'description': """
+This module changes the address format to Korean format.
```

Comment:
```suggestion
Korean Accounting Module
========================

    * changes the address format to Korean format.
```
This module will be used for more than just the format.